### ISSUE TITLE
Shiny data not updated at the same time

### ISSUE THREAD

**Issue by choogland** [dsbbfinddx/FINDCov19TrackerData#28]
When new or updated file of tests data is pushed in FINDCov19TrackerData/raw, cases would be updated but not tests in FINDCov19TrackerData/processed. It happened yesterday for example, and data was not rectified with the scheduled job last night.
See Deploy from Github Actions build 518595729

**Comment by dsbb-finddx** [comment]
I fixed the issue with negative values, coronavirus_test.csv is updated but data_all.csv did not get updated
https://github.com/dsbbfinddx/FINDCov19TrackerData/tree/master/processed

**Comment by pat-s** [comment]
@choogland Appreciated if you could include direct links in such cases, otherwise it is hard to follow which files you refer to since they get updated constantly.

@dsbb-finddx I'll take a look.

I think these are two separate points mentioned here, aren't they? It is not easy to follow along if comments do not relate to the previous one. If in doubt, it would be great if you could just open a separate issue to reduce possible confusion.

**Comment by pat-s** [comment]
> @pat-s I fixed the issue with negative values, coronavirus_test.csv is updated but data_all.csv did not get updated
https://github.com/dsbbfinddx/FINDCov19TrackerData/tree/master/processed

I deployed again and `data_all.csv` looks fine now.

Not quite sure what the issue was - maybe fixing #27 but actually we've been fine before all the time and there were no recent changes.

**Comment by pat-s** [comment]
@choogland 

I am loosing overview a bit because [data_all.csv](https://raw.githubusercontent.com/dsbbfinddx/FINDCov19TrackerData/master/processed/data_all.csv) is not really readable easily and the diffs are not processed due to its size.

I think we should move to the new approach with daily files & diffs as soon as possible since backtracking issues and daily diffs are way more transparent there.

Currently I can't say why on that specific run only cases and not tests were updated.

**Comment by pat-s** [comment]
The shiny app is currently always re-deployed for each update in FindCov19TrackerData.
